### ISSUE TITLE
Server 2016 issue with DismApi.EnableFeatureByPackageName and DismApi.DisableFeature methods

### ISSUE THREAD

**Issue by jonarnett** [josemesona/ManagedDism#19]
Enabling or disabling the IIS-WebServerRole feature on Server 2016 does not seem to work and results in the following error:

```
Object reference not set to an instance of an object.
   at Microsoft.Dism.DismApi.ThrowIfFail(Func`1 func, Int32[] successCodes)
   at Microsoft.Dism.DismApi.DisableFeature(DismSession session, String featureName,
       String packageName, Boolean removePayload, DismProgressCallback progressCallback)
```
Running on Windows 10 and Server 2012R2 does not produce any errors.

This is a code example for the disable :

```
        DismApi.Initialize(DismLogLevel.LogErrors);

        try
        {
            using (DismSession dismSession = DismApi.OpenOnlineSession())
            {
                DismApi.DisableFeature(dismSession, "IIS-WebServerRole", packageName: null,
                    removePayload: false, progressCallback: null);
            }
        }
        catch (Exception ex)
        {
            Console.Error.WriteLine("{0}\r\n{1}", ex.Message, ex.StackTrace);
        }
        finally
        {
            DismApi.Shutdown();
        }
```

This code example for the enable also results in a similar error on Server 2016:

```
        DismApi.Initialize(DismLogLevel.LogErrors);

        try
        {
            using (DismSession dismSession = DismApi.OpenOnlineSession())
            {
                DismApi.EnableFeatureByPackageName(dismSession, "IIS-WebServerRole", packageName: null,
                    limitAccess: false, enableAll: true);
            }
        }
        catch (Exception ex)
        {
            Console.Error.WriteLine("{0}\r\n{1}", ex.Message, ex.StackTrace);
        }
        finally
        {
            DismApi.Shutdown();
        }
```

Perhaps a legitimate error is not being returned because of a null reference in the error handling?

**Comment by jonarnett** [comment]
Even though the enable throws an exception, the call actually seems to succeed.  The disable throws an exception, but does not seem to succeed.

**Comment by jeffkl** [comment]
Believed to be fixed by #22
